_data/accessibility-tests/button-group.yml

```diff
@@ -0,0 +1,85 @@
+title: Button group
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey button meaning.
+    summary_additional: When you view a buttons in a group, you can see and read text, and do not have to rely on color alone for meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.1
+  - summary: Button group content meets color contrast requirements.
+    summary_additional: When you use ANDI color contrast analyzer on a button in a group, the contrast between the text and background color is at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: The width of each button is at least 44 px.
+    summary_additional: When you select "inspect element" to review the button's size, you can see the total width of the button is at least 44 pixels wide.
```

Comment:
recommend changing to "When you select "inspect element" to review a button's size..." - change "the" to "a", as there will be more than one button. "The" implies there is only one. Leave "the" in the second sentence though.

Reply:
Updated in [7ffca7d](https://github.com/uswds/uswds-site/pull/3009/commits/7ffca7d5140778a2c653304845d5082e45ebe2e7)